src/service/action-impl.js

```diff
@@ -121,6 +126,39 @@ export class ActionService {
   }
 
   /**
+   * Installs action handler for the specified element.
+   * @param {!Element} target
+   * @param {function(!ActionInvocation)} handler
```

Comment:
do we need a bang on `function`? (cant remember if function was implicitly not nullable, i know Function definitely is nullable by default)